projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss

```diff
@@ -90,14 +90,62 @@
     @include igx-tree-typography($type-scale);
 
     // Add theme type-scale specific quirks
-    @if ($_variant == 'material' or $_variant == 'fluent') {
+    @if ($_variant == 'material') {
         @include igx-calendar-typography($type-scale);
         @include igx-card-typography($type-scale);
         @include igx-dialog-typography($type-scale);
         @include igx-drop-down-typography($type-scale);
         @include igx-list-typography($type-scale);
     }
 
+    @if ($_variant == 'material' or $_variant == 'fluent') {
+        @include igx-calendar-typography($type-scale);
+        @include igx-dialog-typography($type-scale);
+        @include igx-drop-down-typography($type-scale);
+    }
+
+    @if ($_variant == 'fluent') {
+        @include igx-checkbox-typography($type-scale, $categories: (
+            label: 'body-2'
+        ));
+
+        @include igx-radio-typography($type-scale, $categories: (
+            label: 'body-2'
+        ));
+
+        @include igx-switch-typography($type-scale, $categories: (
+            label: 'body-2'
+        ));
+
+        @include igx-input-group-typography($type-scale, $categories: (
+            input-text: 'body-2'
+        ));
+
+        @include igx-navdrawer-typography($type-scale, $categories: (
+            item: 'body-2',
+            header: 'caption'
+        ));
+
+        @include igx-card-typography($type-scale, $categories: (
+            title: 'subtitle-1',
+            subtitle: 'body-2',
+        ));
+
+        @include igx-list-typography($type-scale, $categories: (
+            item: 'body-1',
+            title: 'caption',
+        ));
```

Comment:
aren't both of these supposed to be caption typescale, I noticed some use of body 1 in the Kit but that looks like a mistake on my side. Where are "item" and "title" exactly displayed?

Reply:
If one or the other was taken from the Custom List item check the updates to the Kit, we don't really use it whatsoever as a symbol but I made it look and feel more like fluent now <img width="655" alt="Screenshot 2022-01-04 at 16 53 05" src="https://user-images.githubusercontent.com/1702064/148069523-dea575d3-3657-44dd-8ff5-f1554f95f922.png">